Feature/Login.feature

```diff
@@ -0,0 +1,12 @@
+﻿
+Feature: Login
+Check if login functionality works
+
+@mytag
+Scenario Outline: Login to website
+	Given I navigate to application and click SignIn
+	And  I enter the login credentials and click login
+		| Email					 | Password           |
+        | rakeshcr48@gmail.com   | anusree			  |

```

Comment:
Don't thing that these are being used. please pass the parameter above

Reply:
These parameters are passed correctly while running the test